etna/ciim/constants.py

```diff
@@ -99,8 +101,8 @@ def __iter__(self):
     [
         Bucket(key="blog", label="Blog posts"),
         Bucket(key="researchGuide", label="Research Guides"),
-        Bucket(key="insight", label="Insights"),
-        Bucket(key="highlight", label="Highlights"),
+        Bucket(key=BucketKeys.INSIGHT.value, label="Insights"),
+        Bucket(key=BucketKeys.HIGHLIGHT.value, label="Highlights"),
         Bucket(key="audio", label="Audio"),
```

Comment:
Hi Allan, maybe we should include those hard coded to the enum, this looks slightly half-done; blog,...

Reply:
Thanks Mo!, Its not half done, the changes here are related to this ticket, since scope of changing constants can cover templates as well I mentioned 
> To look into further to refactor constants using bucket keys

and created DF-470 (as mentioned in the DF-457 )to address this bit.

Reply:
Hi Allan, if we are here, we should it, just to be consistent if reviewed by anybody.

Reply:
> Hi Allan, if we are here, we should it, just to be consistent if reviewed by anybody.

Not sure what you mean.  I've resolved this comment after responding.

Reply:
What I mean we do not need to open another ticket just to resolve something we can fix now since we are here.

Reply:
Its best practice to keep separate. see if you recollect discussion [PR](https://github.com/nationalarchives/ds-wagtail/pull/449)  `{% if form.group.value != 'nonTna' %}` 
The scope is wider `.py, .html` files, so I discussed with PO, team meeting with this consideration went for another ticket. Hope this will be fine with you.

Reply:
In this case can you please create a ticket and assign it to yourself, kindly post the ticket number here.

Reply:
Already mentioned it see above 
> created DF-470 (as mentioned in the DF-457 )to address this bit.